iOS_homework_1.playground/Pages/Fractional values.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,84 @@
+import Foundation
+
+struct Fractional {
+    var numerator: Int
+    var denominator: Int
+    
+    init(numerator: Int, denominator: Int) {
+            self.numerator = numerator
+            self.denominator = denominator
+        }
+    //Функция вывода
+    func printFr() {
```

Comment:
```suggestion
    func printFraction() {
```
не ленись, дописывай